### PR TITLE
dependabot: update config to include CLA: trivial, set branches etc

### PR DESCRIPTION
**Pull request by paulidale**
Not 100% sure the settings are good or not.  Time will tell.

### CODE REVIEW

**Review by t8m — APPROVED** [review]

**Comment by paulidale** [comment]
ping for second review @openssl/committers

**Review by tom-cosgrove-arm — APPROVED** [review]
LGTM

**Comment by t8m** [comment]
Merged to master branch. Thank you.

**Pull request closed without merging**
